### ISSUE TITLE
新参者櫻坂46三期生 bug

### ISSUE THREAD

**Issue by AzureOSK** [proshunsuke/colmsg#98]
There's some sort of bug when running `colmsg --s_refresh_token <refresh_token>`; it gets stuck downloading to the 新参者櫻坂46三期生 folder and never gets to the other members.

![image](https://github.com/proshunsuke/colmsg/assets/35568697/d11ef6c0-9daf-4a83-a6b5-c0391c882768)

As far as I know, the same bug does not occur when downloading to 新参者日向坂46四期生:

![image](https://github.com/proshunsuke/colmsg/assets/35568697/9050bbcb-3fab-4859-82aa-7e8bffac57df)

**Comment by asdf670** [comment]
Create a text file 280652_0_20231109122246.txt in the 新参者日向坂46四期生 folder and try again.

**Comment by AzureOSK** [comment]
Closing as it now works correctly after creating 280652_0_20231109122246.txt in the 新参者日向坂46四期生 folder.

![image](https://github.com/proshunsuke/colmsg/assets/35568697/f0bd98a1-6773-445b-ba04-ee5f42c60374)

**Comment by proshunsuke** [comment]
I've finally found the time to investigate this issue. It appears that message retrieval is stuck in an infinite loop from November 6th to 9th.

Here is the section of the code where the loop is occurring:
[https://github.com/proshunsuke/colmsg/blob/18da205d8069e06c803bcc33134d6ed8a042b631/src/message/saver.rs#L92-L106](https://github.com/proshunsuke/colmsg/blob/18da205d8069e06c803bcc33134d6ed8a042b631/src/message/saver.rs#L92-L106)

The process is as follows:

- Retrieve up to 100 messages.
- If fewer than 100 messages are retrieved, it means we have reached the most recent message, so the loop ends.
- Otherwise, it means there are more new messages, so we retrieve up to 100 messages again.
  - For this retrieval, the `fromdate` is set to the timestamp of the most recent message obtained, and we aim to get messages after this `fromdate`.

At this time, the `updated_at` value returned by the API (to which `fromdate` corresponds) was `2023-11-09T09:50:17Z` for all 100 messages. According to the above logic, the `fromdate` would perpetually be `2023-11-09T09:50:17Z`, resulting in an infinite loop of message retrieval from November 6th to 9th.
